libcodechecker/report.py

```diff
@@ -84,7 +84,7 @@ def compare_ctrl_sections(curr, prev):
         from_col = m_loc.get('col')
         until_col = m_loc.get('col')
 
-        line_content = linecache.getline(source_file, source_line)
+        line_content = get_line(source_file, source_line)
```

Comment:
If the hash is generated on the server side, because the original PLIST does not contain it, then the source_file will point to the path that is only valid on the client's machine.

Either fix this, or put here a fixme and create an issue

Reply:
Before the store do we overwrite the file paths in the plist files, to the directory where they are extracted, or the path changes are only done during the report processing and the actual plist files are not modified?